cas/remote.go

```diff
@@ -44,6 +46,7 @@ func NewRemote(aciurl, sigurl string) *Remote {
 
 type Remote struct {
```

Comment:
Per discussion we will probably need to change this model a bit as we introduce more indexing but this is ok for now.